Makefile

```diff
@@ -1,6 +1,6 @@
 CARGO = cargo
 NEAR  = near
-FEATURES = contract,log
```

Comment:
Just a sidenote, shouldn't this only be contract + exit-precompiles? We should be using log, right?

Reply:
We need logging to know what happens inside the contract. Otherwise we won't have any chance for that

Reply:
I meant more so, on our mainnet deploys, we shouldn't be using `log`. So we need to be careful about that, yes?

Reply:
If we remove log, debugging errors in production will be hell.